### PR TITLE
Stripping empty arguments in --command string.

### PR DESCRIPTION
**Pull request by normoes**
This resolves issue https://github.com/fabfuel/ecs-deploy/issues/90.

### CODE REVIEW

File: ecs_deploy/ecs.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def set_commands(self, **commands):
                     old_value=container.get(u'command')
                 )
                 self._diff.append(diff)
-                container[u'command'] = new_command.split(" ")
+                container[u'command'] = [ command for command in new_command.split(" ") if command ]
```
Comment (mohsinhijazee) [inline ecs_deploy/ecs.py]: Interesting. I forgot to do that in my original PR.

Comment (fabfuel) [reply ecs_deploy/ecs.py]: Please reformat it to follow PEP8 (regarding whitespace and line-length) and we can merge it! Thanks @normoes !

**Comment by fabfuel** [comment]
Oh. Sorry, now we ran in a little conflict here, as @mohsinhijazee refactored the command parsing in #86. 
@normoes, your change now needs to go to the new `parse_command()` method in
https://github.com/fabfuel/ecs-deploy/blob/develop/ecs_deploy/ecs.py#L217

Would you adjust your PR accordingly? I will merge it soon after.

Thanks!
Fabian

**Comment by normoes** [comment]
Yes, I will do that. No problem.

**Comment by normoes** [comment]
@fabfuel 
Done. I rebased to one commit only to make it look nicer ;)

File: ecs_deploy/ecs.py [hunk 2]
```diff
@@ -214,7 +214,7 @@ def parse_command(command):
                     "command: {} resulting in error: {}"
                     .format(command, str(e)))
 
-        return command.split(' ')
+        return [cmd for cmd in command.split(" ") if cmd]
```
Comment (fabfuel) [inline ecs_deploy/ecs.py]: Thanks @normoes !

I was just double checking this. Only removing the explicit empty string (`' '`) will have the same  result, as `split()` then splits "by arbitrary strings of whitespace characters (space, tab, newline, return, formfeed)" ([docs](https://docs.python.org/2/library/string.html#string.split))
 
```python
>>> '  foo  bar '.split()
['foo', 'bar']
```
Do you want to simplify this?

Comment (normoes) [reply ecs_deploy/ecs.py]: @fabfuel Good point. Done.

File: ecs_deploy/ecs.py [hunk 3]
```diff
@@ -214,7 +214,7 @@ def parse_command(command):
                     "command: {} resulting in error: {}"
                     .format(command, str(e)))
 
-        return command.split(' ')
+        return [cmd for cmd in command.split() if cmd]
```
Comment (fabfuel) [inline ecs_deploy/ecs.py]: Hi @normoes, what I meant was this – it's the same result as your line:
```suggestion
        return command.split()
```

Comment (normoes) [reply ecs_deploy/ecs.py]: Now I get it ;)

Comment (fabfuel) [reply ecs_deploy/ecs.py]: Thanks! 🎉

**Pull request merged**